com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -243,6 +243,15 @@ public struct SimulatorParameters
             PacketDropRate = 0
         };
 
+        private struct PacketLossCache
+        {
+            public int PacketsReceived;
+            public int PacketsDropped;
+            public float PacketLoss;
```

Comment:
Does this field need to be stored? I only see it being read immediately after it's written. Could it be a local?